sa/_db/migrations/20170423163751_CreateAuthorizationsTable.sql

```diff
@@ -0,0 +1,30 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE `authorizations` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `token` varchar(255) NOT NULL,
+  `identifierValue` varchar(255) DEFAULT NULL,
+  `identifierType` int DEFAULT NULL,
+  `registrationID` bigint(20) DEFAULT NULL,
+  `status` int DEFAULT 0,
+  `expires` datetime DEFAULT NULL,
+  `combinations` varchar(255) DEFAULT NULL,
```

Comment:
We can kill `combinations`. The latest version of the spec allows you to fulfill exactly one challenge per authz. Instead, we should have an `allowedChallenges` tinyint that serves as a bitmap of challenges permitted for this authz. We can also reuse that once a challenge has actually been fulfilled to indicate which one was fulfilled.